prov/util/src/util_attr.c

```diff
@@ -432,9 +432,9 @@ int ofi_check_domain_attr(const struct fi_provider *prov, uint32_t api_version,
 {
 	if (prov_attr->name && user_attr->name &&
 	    strcasecmp(user_attr->name, prov_attr->name)) {
-		FI_INFO(prov, FI_LOG_CORE, "Unknown domain name\n");
-		FI_INFO_CHECK(prov, prov_attr, user_attr, name,
-			      FI_TYPE_DOMAIN_ATTR);
+		FI_INFO(prov, FI_LOG_CORE, "Unknown domain name\n"
+			"Supported: %s\nExpected: %s\n",
+			prov_attr->name, user_attr->name);
```

Comment:
Sure. Thank you. But it was an incorrect usage of the macro with name field. I guess, the more better will be introduce appropriate common macro for this case to don't forget/misprint in the future.